# Welcome to Lijun Cheng Lab Pages

### Dr.Lijun Cheng works in Department of Biomedical Informatics, College of Medicine, Ohio State University as an assistant professor 
- https://medicine.osu.edu/find-faculty/non-clinical/biomedical-informatics/lijun-cheng
- Contact: 1800 Cannon Drive, Columbus, OH 43210, United State
- Email: lijun.cheng@osumc.edu


## AREA OF EXPERTISE
### Computational precision medicine in big data mining 
**The aim of "precision medicine" is to devise individualized treatment strategies and therapies. Artificial Intelligence (AI) methods play an important role in predicting the efficacy of drugs from clinical study data, based on patient characteristics. Artificial Intelligence (AI) models can calculate potential success of therapeutic targets, biomarkers, and new drugs. Dr.Cheng lab focuses on new AI mathematical methods development that efficiently identify the characteristics that are relevant for such predictions in precision medicine.**
- Identifying subgroups of patients who respond particularly well to the active agent and do not have an elevated risk of side-effects by multi-omics data integration analysis. 
- The biomarkers identification to predict the subgroups of patients for whom treatment with a newly developed drug will be more effective than the standard treatment.
- Molecular mechanisms of drug resistance/disease progression.
- Efficacy therapeutic target and drug identification combating drug resistance in cancer. 

### CONTRIBUTIONs TO SCIENCE

**Dr. Cheng’s complete list of published work can be accessed in MyBibliography and google scholar (Dr.Cheng has published 39 peer reviewed journal papers and 16 proceedings papers and 3 books. She obtained 5 authorized patents and 2 software copyrights. )**

- The full journal publication list can be found in [NCBI MyBibliography Lijun](https://www.ncbi.nlm.nih.gov/myncbi/1lKz2YI8g8M5L/bibliography/public/)

- All papers list on google scholar by [the link](https://scholar.google.com/citations?user=UtDzFE8AAAAJ&hl=en)

### DEVELOPPED SOFTWARE

####	Precision medicine drug response identification, DRPM (https://pcm2019.shinyapps.io/Drug_Response_Prediction/) website platform 
- DRPM is for predicting an individual's response to cancer target therapy by patient gene expression profile. Especially to drug resistance patient, DRPM will recommend optional drug treatment. A novel artificial intelligence modeling (layer optimal pattern matching) between cancer cells and tumors based on both gene expression profiles and drug response to seek optional experimental cells and drugs for individual patients. The DRPM connected the evidence from patients to the basic science experiment to generate a therapeutic hypothesis providing a strong theoretical basis.
####	TrilogPM platform (https://pcm2019.shinyapps.io/TrilogyPM/)  for precision medicine– “the right treatment, for the right patient, at the right time."
- TrilogPM, a comprehensive evidence knowledgebase in precision cancer medicine. It is a Shiny web server currently for searching drug, target, and genome variation (biomarkers) in copy number variation, mutation and fusion, and cancer type and associated drug treatment. The system integrated the most famous six precision medicine database together to provide a comprehensive evidence knowledgebase in precision cancer medicine for clinical practice and clinical trial generate hypothesis. 
- TrilogyPM software is freely available to biomedical researchers and educators in the non-profit sector.
####	SCNrank (https://pcm2019.shinyapps.io/SCNrank/) platform, for therapeutic targets identification 
- A novel artificial intelligence modeling (graph pattern matching and ranking algorithm) with a Shiny web server for potential cancer target identification, target molecular mechanism, and visualization. The SCNrank system is datamining on big data to seek cancer potential targets, including expression profiles from tumor tissue, adjacent normal tissue, and cell-line; protein-protein interaction network (PPI); and CRISPR-Cas9 gene knock-out from cancer cell data. By gene functions variations of cancer cell lines from genome scale CRISPR-Cas9, SCNrank matches its functions with tumors to guide precision cancer medicine.
####	XDeath (https://pcm2019.shinyapps.io/LijunLab_Pathway_Enrichment/) an interactive platform based on shinyapps cloud. 
- XDeath is to identify the most significant therapeutic targets to induce or program cell death and keep with limit cell growth. At the same time, the network model is for the mechanism interpreting of disease progression and drug resistance.
- Using a deep neural network model, XDeath, is used to precisely identify therapeutic targets that are associated with cancer initiation and/or progression. The unique method could detect molecular targeting of distinct deregulated active signaling elements that might contribute to their sustained growth, survival, and treatment resistance, therefore, is of immense therapeutic interest. These novel target identification approaches should improve the efficacy of current therapeutic treatments against highly aggressive, metastatic, recurrent, and lethal cancers. Twenty-nine pathways of cell death with eight cell death modes, apoptosis, autophagy, autosis, immunogenic (T cell, B cell), necroptosis, the broad-spectrum protein kinase C (PKC) ferroptosis, and proliferation (survival) are observed systematically. Molecular mechanisms of crosstalk during cancer progresses dynamically from the early stage to the advanced stage and is investigated in two independent datasets, TCGA: 550 patients for testing, and GEO serial ID GSE21032: 177 patients for validation.
####	XDeathDB (https://pcm2019.shinyapps.io/XDeathDB/) is a website on shinyapps cloud. 
- XDeathDB creates a comprehensive search engine of each mode of cell death and their interaction at the gene and pathway relevant to regulation of cell death. With this platform, users can search specific interactions from vast interdependent sub-networks that occur in the realm of cell death, including any genes, any pathways and 12 cell death modes. 
- XDeathDB is a dynamic interactive system. Users can upload gene-expression profiles linked with phenotypes and create their own networks using their own genes of interest. In addition, users can import dynamic networks from a txt file directly and export dynamic networks to a txt file for further analysis. 
- XDeathDB implemented dynamic network construction method in a modular way and allow users to freely select and combine these modules to obtain their own network construction.

####	XDeathDB (https://pcm2019.shinyapps.io/XDeathDB/) is a website on shinyapps cloud. 

### STUDENTS
**Involvement in graduate/professional exams, theses, and dissertations and undergraduate research**
- **Undergraduate Students**: 180 students per year from year 1998 to year 2008; 21 students in year 2017; 18 students in year 2018;
- **Masters Students:** Pooja Chandra, Sai Mounika Inavolu, Varshini Vasudevaraja, Aniruddha Vikram Pawar
- **Doctoral students:** Enze Liu, Aida Yazdanparast 
- **Post-doctoral Fellow:** Abhishek Majumdar, Tao Han
