# Welcome to Lijun Cheng Lab Pages
![Framework](./Lijun_photo.jpg)
- This repository houses the Lijun Lab website, which is shared among members through Github and hosted by Dr.Lijun Cheng.
- Lab members should keep their own pages current, as well as contribute to the lab news feed and update research themes and study systems as appropriate.

# Lijun Cheng Ph.D
### 01/2018-, Assistant professor, Department of biomedical informatics, College of medicine, Ohio state university (OSU)
- [Introduction on OSU webpage](https://medicine.osu.edu/find-faculty/non-clinical/biomedical-informatics/lijun-cheng)
- Contact: 1800 Cannon Drive, Columbus, OH 43210, United State
- Email: lijun.cheng@osumc.edu

## Latest news
“Co-targeting Plk1 and DNMT3a in advanced prostate cancer” (Research Article, No. advs.202101458R1, <b> High Impact Factor of 16 </b>) published on Advanced Science as co-first author.

### PREVIOUS POSITIONS
- 05/2015 – 12/2017 	Assistant Researcher Professor, Department of Medical and Molecular Genetics, Indiana University, Indianapolis, Indiana, U.S.
- 04/2014 – 05/2015 	Postdoctoral Fellow, Department of Medical and Molecular Genetics, Indiana University, Indianapolis, IN, U.S.
- 12/2012 – 03/2014 	Assistant Researcher, Shanghai Jiao Tong University, State Key Laboratory of Oncogenes and Related Genes, Shanghai Cancer Institute, P. R. China
- 07/2008 – 11/2012   Donghua University, P.R. China, and Concordia University, Canada, Joint Ph.D. , Artificial Intelligence Pattern Identification, Electrical Engineering and Computer Science

## AREA OF EXPERTISE
### Computational precision medicine by artifical intelligence big data mining 
**The aim of "precision medicine" is to devise individualized treatment strategies and therapies. Artificial Intelligence (AI) methods play an important role in predicting the efficacy of drugs from clinical study data, based on patient characteristics. AI models can calculate potential success of therapeutic targets, biomarkers, and new drugs. Dr.Cheng lab focuses on new AI mathematical methods development that efficiently identify the characteristics that are relevant for such predictions in precision medicine.**
- Identifying subgroups of patients who respond particularly well to the active agent and do not have an elevated risk of side-effects by multi-omics data integration analysis. 
- The biomarkers identification to predict the subgroups of patients for whom treatment with a newly developed drug will be more effective than the standard treatment.
- Molecular mechanisms of drug resistance/disease progression.
- Efficacy therapeutic target and drug identification combating drug resistance in cancer. 

### DEVELOPPED SOFTWARE
####	[Precision Medicine System, DRPM](https://pcm2019.shinyapps.io/Drug_Response_Prediction/) interactive website platform 
- DRPM is for predicting an individual's response to cancer target therapy by patient gene expression profile. To drug resistance patient, DRPM will recommend optional drug treatment.
- A novel artificial intelligence modeling (layer optimal pattern matching) between cancer cells and tumors based on both gene expression profiles and drug response to seek optional experimental cells and drugs for individual patients. 
- The DRPM connected the evidence from patients to the basic science experiment to generate a therapeutic hypothesis providing a strong theoretical basis.
- [User evaluation and feedback, usability surveys to DRPM](https://osu.az1.qualtrics.com/jfe/form/SV_24D9IO4eRc6lQnb)
####	[TrilogPM platform](https://pcm2019.shinyapps.io/TrilogyPM/)  for precision medicine– “the right treatment, for the right patient, at the right time."
- TrilogPM, a comprehensive evidence knowledgebase in precision cancer medicine. It is a Shiny web server currently for searching drug, target, and genome variation (biomarkers) in copy number variation, mutation and fusion, and cancer type and associated drug treatment. The system integrated the most famous six precision medicine database together to provide a comprehensive evidence knowledgebase in precision cancer medicine for clinical practice and clinical trial generate hypothesis. 
- TrilogyPM software is freely available to biomedical researchers and educators in the non-profit sector.
- [User evaluation and feedback, usability surveys to TrilogyPM](https://osu.az1.qualtrics.com/jfe/form/SV_4OrqJwO9vq5P28R)
####	[SCNrank](https://pcm2019.shinyapps.io/SCNrank/) platform, for therapeutic targets identification in cancer by a large screening CRISPR-Cas9 guidance
- A novel artificial intelligence modeling (graph pattern matching and ranking algorithm) with a Shiny web server for potential cancer target identification, target molecular mechanism, and visualization.
- The potential identified therapiutic targets and target molecular mechanism translation have been connected to clinical trails successfully in ten cancer types, colon adenocarcinoma, esophageal carcinoma, head and neck squamous cell carcinoma, kidney renal clear cell carcinoma, kidney renal papillary cell carcinoma, liver hepatocellular carcinoma, lung adenocarcinoma, lung squamous cell carcinoma, pancreatic adenocarcinoma, thyroid carcinoma.
- The SCNrank system is datamining on big data to seek cancer potential targets, including expression profiles from tumor tissue, adjacent normal tissue, and cell-line; protein-protein interaction network (PPI); and CRISPR-Cas9 gene knock-out from cancer cell data. By gene functions variations of cancer cell lines from genome scale CRISPR-Cas9, SCNrank matches its functions with tumors to guide precision cancer medicine.
- [User evaluation and feedback, usability surveys to SCNrank](https://osu.az1.qualtrics.com/jfe/form/SV_cN4aWPI7wU7IecR)
####	[XDeath](https://pcm2019.shinyapps.io/LijunLab_Pathway_Enrichment/) for therapeutic target identification to induce cell death--- an interactive platform based on shinyapps cloud. 
- XDeath is to identify the most significant therapeutic targets to induce cell death regulately. At the same time, the XDeath model is for the mechanism interpreting of disease progression and drug resistance.
- Using a deep neural network model, XDeath, is used to precisely identify therapeutic targets that are associated with cancer initiation and/or progression. The unique method could detect molecular targeting of distinct deregulated active signaling elements that might contribute to their sustained growth, survival, and treatment resistance, therefore, is of immense therapeutic interest. These novel target identification approaches should improve the efficacy of current therapeutic treatments against highly aggressive, metastatic, recurrent, and lethal cancers. Twenty-nine pathways of cell death with eight cell death modes, apoptosis, autophagy, autosis, immunogenic (T cell, B cell), necroptosis, the broad-spectrum protein kinase C (PKC) ferroptosis, and proliferation (survival) are observed systematically. Molecular mechanisms of crosstalk during cancer progresses dynamically from the early stage to the advanced stage and is investigated in two independent datasets. Current platform, as prostate cancer for example, we demonstrated how these survival pathways crosstalk switches and caused prostate cancer progression. The optimal therapeutic targets are identified to stop progression, where TCGA: 550 patients for modelling construction, and GEO serial ID GSE21032: 177 patients for validation.
- [User evaluation and feedback, usability surveys to XDeath](https://osu.az1.qualtrics.com/jfe/form/SV_2r8307kGQGELyf3) 
####	[XDeathDB](https://pcm2019.shinyapps.io/XDeathDB/)-- an intertact website on shinyapps cloud for programmed cell death and crosstalk search engine
- XDeathDB creates a comprehensive search engine of molecular mechanisms of cell death and cell death interactions at the key therapeutic targets, cell death hallmark genes and pathway relevant to regulation of cell death. 
- XDeathDB includes 12 cell death modes and 498 pathways in cell cycle, immunology, autophagy, ferroptosis, appoptosis, necrosis, DNA damage, mitochondria, pyroptosis, lysozomal cell death, mitotic cell death, autosis, autophagy, which refer to latest [nature literature molecular mechanisms of cell death](https://www.nature.com/articles/s41418-017-0012-4). 
- With XDeathDB platform, users can search specific interactions from vast interdependent sub-networks that occur in the realm of cell death, including any genes, any pathways and 12 cell death modes. 
- XDeathDB is a dynamic interactive system. Users can upload gene-expression profiles linked with phenotypes and create their own networks using their own genes of interest. In addition, users can import dynamic networks from a txt file directly and export dynamic networks to a txt file for further analysis. 
- XDeathDB implemented dynamic network construction method in a modular way and allow users to freely select and combine these modules to obtain their own network construction.
- [User evaluation and feedback, usability surveys to XDeathDB](https://osu.az1.qualtrics.com/jfe/form/SV_4VflPI9ItadcfWZ)

#### [DGCyTOF](https://github.com/lijcheng/DGCyTOF)
- Mass cytometry, or CyTOF (Fluidigm),  is a novel platform for high-dimensional phenotypic and functional analysis of single cells.
- CyTOF is a variation of flow cytometry in which antibodies are labeled with heavy metal ion tags rather than fluorochromes.
- A new tool Deep learning with Graphical clustering, called DGCyTOF, is developped to identify new cell population by CyTOF big data analysis.

### STUDENTS
- **Involvement in graduate/professional exams, theses, and dissertations and undergraduate research**
- **Undergraduate Students**: 180 students per year from year 1998 to year 2008; 21 students in year 2017; 18 students in year 2018;
- **Masters Students:** Pooja Chandra, Sai Mounika Inavolu, Varshini Vasudevaraja, Aniruddha Vikram Pawar
- **Doctoral students:** Enze Liu, Aida Yazdanparast 
- **Post-doctoral Fellow:** Abhishek Majumdar, Tao Han


### CONTRIBUTIONs TO SCIENCE

**Dr. Cheng’s complete list of published work can be accessed in MyBibliography and google scholar (Dr.Cheng has published 39 peer reviewed journal papers and 16 proceedings papers and 3 books. She obtained 5 authorized patents and 2 software copyrights. )**

- The full journal publication list can be found in [NCBI MyBibliography Lijun](https://www.ncbi.nlm.nih.gov/myncbi/1lKz2YI8g8M5L/bibliography/public/)

- All papers list on google scholar by [the link](https://scholar.google.com/citations?user=UtDzFE8AAAAJ&hl=en)

### AREA OF EXPERTISE PAPER

**Dr.Cheng contribution to science mainly exists in three aspects (list typical 4 papers):**

#### 1.	Artificial intelligence (AI) graph neural network and learning models for pattern recognition tasks
**Dr.Cheng has developed several graph neural network and learning models and applying them in primary open-angle glaucoma identification by dynamic images analysis, therapeutic target identification from membrane protein interaction networks , and predictive biomarkers identification to chemotherapy response in sarcoma patients. Influence of findings: these AI methods led to a new theory system in machine learning and make these identification task speed fast and accuracy improvement sharply in big data mining. All these finding is published on top tie artificial neural network and learning journals. 1.) IEEE Transaction on neural network and learning systems (impact factor, 11.683). 2.) Neurocomputing (impact factor, 3.317), 3) BMC Medicine Genomics (impact factor, 3.17).**
- [1] Yongsheng Ding*, **Lijun Cheng**, Witold Pedrycz, and Kuangrong Hao, Global nonlinear kernel prediction for large dataset with a particle swarm optimized interval support vector regression, IEEE Transaction on neural network and learning systems, 2015, 26(10):2521-2534. (**Co-first authors**)
- [2] Yongsheng Ding, Yizhen Shen, Lihong Ren, and **Lijun Cheng**, Dynamic and collective analysis of membrane protein interaction network based on gene regulatory network model, Neurocomputing, 2012, 98(3):151-158.
- [3] **Lijun Cheng**, Yongsheng Ding*, Hao Kuangrong, and Yifan Hu. An ensemble kernel classifier with immune clonal selection algorithm for automatic discriminant of primary open-angle glaucoma. Neurocomputing. 2012; 83(15):1-11.
- [4] **Lijun Cheng**, Pankita H Pandya, Enze Liu, Pooja Chandra, Limei Wang, Mary E Murray, Jacquelyn Carter, Michael Ferguson, Mohammad Reza Saadatzadeh, Khadijeh Bijangi-Visheshsaraei, Mark Marshall, Lang Li*, Karen E Pollok*, and Jamie L Renbarger*. Integration of genomic copy number variations and chemotherapy-response biomarkers in pediatric sarcoma. BMC Medicine Genomics. 2019, 12(Suppl 1):23. 
#### 2.	Optimal drug-combination identification
**Dr.Cheng developed system biology models to identify effective drug combination treatment for these patients with resistance and metastasis by signaling pathway rewiring mechanism. Her calculational model system got extensive validation and application in different collaborative universities, such as Indiana University and Kentucky University in United State. These studies published on the top tie cancer journals, such as the Cancer Research (impact factor 9.8), Oncogene (impact factor, 7.97), Cancer (impact factor, 5.742), the Journal of Biological Chemistry (impact factor 4.238). The selected paper associated to the research lists as the following:**
- [1] Pankita H Pandya*, **Lijun Cheng***, M Reza Saadatzadeh, Khadijeh Bijangi-Vishehsaraei, Shan Tang, Anthony L Sinn, Melissa A Trowbridge, Kathryn L Coy, Barbara J Bailey, Courtney N Young, Jixin Ding, Erika A Dobrota, Savannah Dyer, Adily Elmi, Quinton Thompson, Farinaz Barghi, Jeremiah Shultz, Eric A Albright, Harlan E Shannon, Mary E Murray, Mark S Marshall, Michael J Ferguson, Todd E Bertrand, L Daniel Wurtz, Sandeep Batra, Lang Li, Jamie L Renbarger, Karen E Pollok. Systems biology approach identifies prognostic signatures of poor overall survival and guides the prioritization of novel BET-CHK1 combination therapy for osteosarcoma. Cancers 2020, 12(9): 2426. (**Co-first authors**). (This study illustrates the MYC signaling pathway switching mechanism with RAD21 signaling pathway and seek BET-CHK1 combination drug treatment to overcome the ‘rewiring’)
- [2] Yifan Kong, **Lijun Cheng**, Fengyi Mao, Zhuangzhuang Zhang, Yanquan Zhang, Elia Farah, Jacob Bosler, Yunfeng Bai, Nihal Ahmad, Shihuan Kuang, Lang Li and Xiaoqi Liu. Inhibition of cholesterol biosynthesis overcomes enzalutamide resistance in castration-resistant prostate cancer (CRPC). The Journal of Biological Chemistry, 293:14328-14341. (This study illustrates the cholesterol signaling pathway switching mechanism with AR signaling pathway and seek combination drug treatment to overcome the temporal ‘rewiring’.)
- [3] Zhuangzhuang Zhang, **Lijun Cheng**, Jie Li, Elia Farah, Nadia M Atallah, Pete E Pascuzzi, Sanjay Gupta, Xiaoqi Liu. Inhibition of the Wnt/β-catenin pathway overcomes resistance to enzalutamide in castration-resistant prostate cancer. Cancer research. 2018, 78 (12): 3147-3162. (This study illustrates the Wnt/ β-catenin pathway switching mechanism with AR signaling pathways and seek new drug combination to overcome the Wnt pathway activate.)
- [4] Jin Li, Yang Huo, Xue Wu, Enze Liu, Zhi Zeng, Zhen Tian, Kunjie Fan, Daniel Stover, **Lijun Cheng**, Lang Li*. Essentiality and transcriptome-enriched pathway scores predict drug-combination synergy. Biology, 2020, 9(9):278-291. (This study is to qualify analysis crosstalk of pathway to predict drug combination.)
- [5] Jinghui Liu, Daheng He, **Lijun Cheng**, Changkun Huang, Yanquan Zhang, Xiongjian Rao, Yifan Kong, Chaohao Li, Zhuangzhuang Zhang, Jinpeng Liu, Karrie Jones, Dana Napier, Eun Y Lee, Chi Wang, Xiaoqi Liu. p300/CBP inhibition enhances the efficacy of programmed death-ligand 1 blockade treatment in prostate cancer. Oncogene. 2020, 39:3939-3951. (This study illustrates the cell death immunology pathway switching mechanism with AR signaling pathways and seek new drug combination to overcome the new pathway activate.)
#### 3. Drug repurpose: optimal therapeutic targets identification 
**Dr.Cheng has developed several system biology optimization methods for therapeutic targets identification and associated effective drug recommendation for cancer. Systematic network analysis on cancer patients has multiple potential biological and clinical applications. A better understanding of the effects of gene/protein interaction may lead to the identification of cancer genes and correlated pathways, which, in turn, may offer better targets for drug development in cancer treatment. An optimizing subnetwork searching of drug targets algorithms is developed for drug targets selection, such as IODNE for triple negative breast cancer and SCNrank for pancreatic ductal adenocarcinoma. Associated study results are published on the top tie biomedicial journals, the Journal of the American Medical Informatics Association (impact factor, 4.112), BMC Medicine Genomics (impact factor, 3.17) and Genes (impact factor, 2.984).**
- [1] Sai Mounika Inavolu, Milan Radovich, Varshini Vasudevaraja, Kinnebrew, Garrett Hess, Shijun Zhang, Jamie Renbarger, and **Lijun Cheng***, IODNE: An Integrated Optimization method for identifying the Deregulated sub-NEtwork for precision medicine in cancer, CPT: Pharmacometrics & Systems Pharmacology, 2017, 6(3):168-176. 
- [2] Enze Liu, Zhuang Zhuang Zhang, Xiaolin Cheng, Xiaoqi Liu, and **Lijun Cheng***, SCNrank: spectral clustering for network-based ranking to reveal potential drug-targets and its application in pancreatic ductal adenocarcinoma. BMC Med Genomics. 2020,13(Suppl 5):50. 
- [3] **Lijun Cheng**, Bryan P. Schneider and Lang Li*, A Bioinformatics approach for the precision medicine off-label drug usage among triple negative breast cancer patients, the Journal of the American Medical Informatics Association, 2016, pii: ocw004.
- [4] **Lijun Cheng**, Abhishek Majumdar, Daniel Stover, Shaofeng Wu, Yaoqin Lu, Lang Li. Computational cancer cell models to guide precision breast cancer medicine. Genes (Basel). 2020, 11(3):263. 
